backend/elasticsearch/src/main/java/org/hibernate/search/backend/elasticsearch/logging/impl/Log.java

```diff
@@ -329,4 +329,8 @@ SearchException conflictingFieldTypesForProjection(String absoluteFieldPath,
 	@Message(id = ID_OFFSET_3 + 48,
 			value = "Sorting is not enabled for field '%1$s'. Make sure the field is marked as sortable.")
 	SearchException unsortableField(String absoluteFieldPath, @Param EventContext context);
+
+	@Message(id = ID_OFFSET_3 + 49,
+			value = "Incompatible id converters: <%1$s> <%2$s>")
+	SearchException incompatibleIdConverters(String idConverter1, String idConverter2);
```

Comment:
I suspect this message is not consistent with the message used for incompatible field types; I'll check that.